src/components/BlogComponents/Pagination/index.module.scss

```diff
@@ -0,0 +1,62 @@
+.pagination {
+  display: flex;
+  flex-wrap: wrap;
+  list-style: none;
+  margin: 0;
+  padding: 0;
+
+  .pagination-page {
+    align-items: center;
+    background-color: var(--brand0);
+    color: var(--black9);
+    cursor: pointer;
```

Comment:
I wonder about the accessibility, if we're using `cursor` pointer, does it mean the component is not an actual link component? Or what? 🤔